### PR TITLE
FEATURE: Add a pagination for exceptions

### PR DESCRIPTION
**Pull request by Kleisli**
If there are many exceptions, it takes quite some time to parse them all, but in the end you still only need the most recent.

Now it only loads the 10 most recent exceptions, but you can rise the limit to load more.

### CODE REVIEW

File: Classes/Controller/LogsController.php [hunk 1]
```diff
@@ -56,7 +56,7 @@ class LogsController extends AbstractModuleController
     /**
      * Renders the app to interact with the nodetype graph
      */
-    public function indexAction(): void
+    public function indexAction(int $limit = 10): void
```
Comment (Sebobo) [inline Classes/Controller/LogsController.php]: How about we introduce an actual pagination here. So like
```suggestion
    public function indexAction(int $page = 0): void
```

And read the `limit` from the Settings yaml?

**Comment by Sebobo** [comment]
Thx! Makes sense :)
But I would directly try to turn this into a proper pagination already then or at least prepare for it. Wdyt?

**Comment by Kleisli** [comment]
I updated the PR now with a proper pagination for Exceptions.

And I also reversed the order of log entries in the LogFile view, because usually I'm interested in the most recent entries. If you don't like it, I could also wrap that part with a configurable setting.

**Comment by Sebobo** [comment]
Thx, will check it out in the next days. I also have a branch with some changes regarding the file loading as I was able to make it quite a bit faster.

You used some Neos 7.1 features. So I pushed an update to composer.json to require Neos 7.3 which I anyway wanted to do.

**Pull request merged**